### DISCUSSION TITLE
Package.json file missing!!!!!

### DISCUSSION THREAD

**Discussion by RudraKumarSharma** [Amna-Hassan04/Serenity-Guide#25] [General]
the repo is misisng package.json file due to which  npm install command is failing
![Uploading Screenshot 2024-10-02 at 11.26.42 AM.png…]()

**Comment by Amna-Hassan04** [comment]
you can raise this an issue. also the screenshot is not showing.

**Reply by RudraKumarSharma** [reply]
But this issue can inly be resolved by you as package.json file is provided by the owner of repo

**Reply by Amna-Hassan04** [reply]
ok let me see.

**Comment by Amna-Hassan04** [comment]
@RudraKumarSharma I have replied to you in the issue comments, hope you were able to get the project working.
